### PR TITLE
fontconfig: drop noarch

### PR DESCRIPTION
**Pull request by fosslinux**

### CODE REVIEW

File: srcpkgs/fontconfig/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'fontconfig'
 pkgname=fontconfig
 version=2.13.1
-revision=2
+revision=3
```
Comment (abenson) [inline srcpkgs/fontconfig/template:4]: Do we want to revbump this or just make the change and wait for it to happen through attrition?

Comment (abenson) [reply srcpkgs/fontconfig/template:4]: Same question for mkfontscale

Comment (fosslinux) [reply srcpkgs/fontconfig/template:4]: Why not they're both small packages. Also we've been dropping noarch through revbumps for quite some time now...

https://github.com/void-linux/void-packages/pull/25929/commits/84af860c6256d2d6e7b3c63080abd8122d86c5f3 and similar in that PR
https://github.com/void-linux/void-packages/pull/25710
https://github.com/void-linux/void-packages/pull/25413

If I am understanding your question correctly...

Comment (Chocimier) [reply srcpkgs/fontconfig/template:4]: @abenson Yes, goal is to get rid of noarch packages from binary repos, and to do that in finite time revbumping them all is required.

**Pull request merged**